tools/js-sdk-release-tools/src/common/changlog/modifyChangelogFileAndBumpVersion.ts

```diff
@@ -56,14 +57,35 @@ function changePackageJSON(packageFolderPath: string, packageVersion: string) {
 function changeClientFile(packageFolderPath: string, packageVersion: string) {
     const packageJsonData: any = JSON.parse(fs.readFileSync(path.join(packageFolderPath, 'package.json'), 'utf8'));
     const packageName = packageJsonData.name.replace("@azure/", "");
-    const files: string[] = fs.readdirSync(path.join(packageFolderPath, 'src'));
-    files.forEach(file => {
-        if (file.endsWith('.ts')) {
-            const data: string = fs.readFileSync(path.join(packageFolderPath, 'src', file), 'utf8');
-            const result = data.replace(/const packageDetails = `azsdk-js-[0-9a-z-]+\/[0-9.a-z-]+`;/g, 'const packageDetails = `azsdk-js-' + packageName + '/' + packageVersion + '`;');
-            fs.writeFileSync(path.join(packageFolderPath, 'src', file), result, 'utf8');
-        }
-    })
+    const sdkType = getSDKType(packageFolderPath)
+    if(sdkType === "HighLevelClient"){ // Update version in src for HLC
+        const files: string[] = fs.readdirSync(path.join(packageFolderPath, 'src'));
+        files.forEach(file => {
+            if (file.endsWith('.ts')) {
+                const data: string = fs.readFileSync(path.join(packageFolderPath, 'src', file), 'utf8');
+                const result = data.replace(/const packageDetails = `azsdk-js-[0-9a-z-]+\/[0-9.a-z-]+`;/g, 'const packageDetails = `azsdk-js-' + packageName + '/' + packageVersion + '`;');
+                fs.writeFileSync(path.join(packageFolderPath, 'src', file), result, 'utf8');
+            }
+        })
+    } else if (sdkType === "ModularClient"){ // Update version in src for Modular
+        const files: string[] = fs.readdirSync(path.join(packageFolderPath, 'src',"api"));
+        files.forEach(file => {
+            if (file.endsWith('Context.ts')) {
+                const data: string = fs.readFileSync(path.join(packageFolderPath, 'src',"api", file), 'utf8');
+                const result = data.replace(/const userAgentInfo = `azsdk-js-[0-9a-z-]+\/[0-9.a-z-]+`;/g, 'const userAgentInfo = `azsdk-js-' + packageName + '/' + packageVersion + '`;');
+                fs.writeFileSync(path.join(packageFolderPath, 'src','api', file), result, 'utf8');
+            }
+        })
+    } else{ // Update version in src for RLC
```

Comment:
please use 
`switch case` block, default should be
```
        default:
            logger.warn(`Unsupported SDK type ${sdkType} to get detact api version`);
            return ApiVersionType.None;
```
to avoid bugs when new type added in the future.